Model/src/GamePackage/Die.java

```diff
@@ -0,0 +1,53 @@
+package GamePackage;
+
+import java.util.Random;
+
+/*****************************************************************
+ The Die Class will be a random number generator that "rolls"
+ a die some methods to return the value of the die.
+
+ @author Dylan Kernohan
+ @version Winter 2018

```

Comment:
The version should have the whole date

Reply:
I was going off the style guide, their example had it like this. We can change it though, not a big deal.